packages/story-editor/src/components/floatingMenu/elements/borderRadius.js

```diff
@@ -17,21 +17,63 @@
 /**
  * External dependencies
  */
-import { Icons } from '@googleforcreators/design-system';
+import { Icons, NumericInput } from '@googleforcreators/design-system';
 import { __ } from '@googleforcreators/i18n';
+import styled from 'styled-components';
 
 /**
  * Internal dependencies
  */
-import { IconButton, Separator } from './shared';
+import { useStory } from '../../../app';
+import { canSupportMultiBorder } from '../../../masks';
+import { DEFAULT_BORDER_RADIUS } from '../../panels/design/sizePosition/radius';
+import { Separator, useProperties } from './shared';
+
+const StyledInput = styled(NumericInput)`
```

Comment:
By the way, I was very happy to see that our existing input component was perfectly styled for this already and just working seemlessly in the menu. I don't know if an input in a menu should be marked up in a particular way, but don't think so. It's not uncommon to have these inputs in a menu (for text editors), so I think it's semantically fine, but there might be some details around focus, that we should add (e.g. pressing arrow keys left-right should move focus, whereas up-down changes the value).